opendbc/car/toyota/carstate.py

```diff
@@ -190,7 +191,10 @@ def update(self, can_parsers) -> structs.CarState:
     if self.CP.carFingerprint in (TSS2_CAR - RADAR_ACC_CAR):
       # distance button is wired to the ACC module (camera or radar)
       prev_distance_button = self.distance_button
-      self.distance_button = cp_acc.vl["ACC_CONTROL"]["DISTANCE"]
+      if self.CP.carFingerprint in (SECOC_CAR):
+        self.distance_button = cp.vl["PCM_CRUISE_4"]["DISTANCE"]
+      else:
+        self.distance_button = cp_acc.vl["ACC_CONTROL"]["DISTANCE"]
```

Comment:
Per our Discord discussions, this signal is inconsistent with openpilot's Driving Personality and gets de-synced sometimes. I suggest to exclude SecOC platforms from parsing the distance button in this PR until a consistent signal is found.

```suggestion
      self.distance_button = cp_acc.vl["ACC_CONTROL"]["DISTANCE"]
```